apps/condo/pages/auth/register.tsx

```diff
@@ -1,473 +1,297 @@
-// @ts-nocheck
-/** @jsx jsx */
-import { css, jsx } from '@emotion/core'
-import { QuestionCircleOutlined } from '@ant-design/icons'
-
-import { TopMenuOnlyLayout } from '@condo/domains/common/components/containers/BaseLayout'
-import { EMAIL_ALREADY_REGISTERED_ERROR, MIN_PASSWORD_LENGTH_ERROR } from '@condo/domains/common/constants/errors'
-import {
-    AUTH as firebaseAuth,
-    initRecaptcha,
-    IS_FIREBASE_CONFIG_VALID, resetRecaptcha,
-} from '@condo/domains/common/utils/firebase.front.utils'
-import { runMutation } from '@condo/domains/common/utils/mutations.utils'
-import { getQueryParams } from '@condo/domains/common/utils/url.utils'
-
-import { getClientSideSenderInfo } from '@condo/domains/common/utils/userid.utils'
-import { REGISTER_NEW_USER_MUTATION } from '@condo/domains/user/gql'
-import { useMutation } from '@core/next/apollo'
-import { useAuth } from '@core/next/auth'
 import { useIntl } from '@core/next/intl'
-import { Alert, Button, Checkbox, Col, Form, Input, Row, Tooltip, Typography } from 'antd'
-import Head from 'next/head'
+import { Col, Form, Input, Row, Typography, Tooltip } from 'antd'
+import { Button } from '@condo/domains/common/components/Button'
+import AuthLayout from '@condo/domains/common/components/containers/BaseLayout/AuthLayout'
 import Router from 'next/router'
-import { createContext, useContext, useEffect, useRef, useState } from 'react'
-
-const AuthContext = createContext({})
-
-const Auth = ({ children, initialUser }) => {
-    const [user, setUser] = useState(initialUser || null)
-
-    useEffect(() => {
-        return firebaseAuth().onAuthStateChanged(async function (user) {
-            if (user) {
-                user.token = await user.getIdToken(true)
-            }
+import { QuestionCircleOutlined } from '@ant-design/icons'
+import React, { useEffect, useState } from 'react'
+import { FormattedMessage } from 'react-intl'
+import { formatPhone } from '@condo/domains/common/utils/helpers'
+import MaskedInput from 'antd-mask-input'
 
-            setUser(user)
-        })
-    }, [])
+const INPUT_STYLE = { width: '273px' }
+const POLICY_LOCATION = '/policy.pdf'
```

Comment:
discuss with PRODUCT

Reply:
now it's a link to an empty file. I Will put the good file to the repository later